spec/integration/blueprint_updates_spec.rb

```diff
@@ -4,7 +4,7 @@
   before(:all) do    
     @client = ConnectionHelper.create_default_connection
     @blueprint = GoodData::Model::ProjectBlueprint.from_json('./spec/data/blueprints/test_project_model_spec.json')
-    @project = @client.create_project_from_blueprint(@blueprint, auth_token: 'stgdss')    
+    @project = @client.create_project_from_blueprint(@blueprint, :auth_token => ENV['GD_PROJECT_TOKEN'])
```

Comment:
can you point me at some place where I can see how do you run the tests? (+ it's still missing to use TESTING in tests - correct?)